plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Definition.hs

```diff
@@ -573,6 +574,27 @@ test_Integer = testNestedM "Integer" $ do
     evals False LessThanEqualsInteger [] [cons @Integer 4001, cons @Integer 4000]
     evals True EqualsInteger [] [cons @Integer (-101), cons @Integer (-101)]
     evals False EqualsInteger [] [cons @Integer 0, cons @Integer 1]
+    for_ [DivideInteger, QuotientInteger, ModInteger, RemainderInteger] $ \ b ->
+        fails (lowerInitialChar $ show b <> "-div-by-zero") b [] [cons @Integer 1, cons @Integer 0]
+    test_ExpModInteger
+
+test_ExpModInteger :: TestNested
+test_ExpModInteger = testNestedM "ExpMod" $ do
```

Comment:
I'm not objecting to these tests (more tests are better!) but maybe there's a discussion to be had about whether we need unit tests like this (which we have for lots of other builtins as well) when we also have the conformance tests.  I suppose these get run more often than the conformance tests though.